TLM/TLM/State/Flags.cs

```diff
@@ -137,6 +137,11 @@ public class Flags {
                 return false;
             }
 
+            if ((Singleton<NetManager>.instance.m_nodes.m_buffer[nodeId].m_flags &
+                NetNode.Flags.Untouchable) != NetNode.Flags.None) {
+                return false;
+            }
+
             ItemClass connectionClass = Singleton<NetManager>
```

Comment:
its just a minor thing but I am not sure if compiler will optimize that second call away ... I'd get the node first and refuse that in both cases.